### PR TITLE
Improve documentation of sample project - #438

### PR DESCRIPTION
**Pull request by arturbosch**
I hate maven. Can't create a pom file easily within minutes without errors. Don't want to waste my time.
When someone requests one, we can open an issue for that and wait for contribution of one ...

### CODE REVIEW

File: detekt-sample-extensions/build.gradle [hunk 1]
```diff
@@ -1,13 +1,40 @@
-configurations {
-	implementation.extendsFrom kotlinImplementation
-	testImplementation.extendsFrom kotlinTest
+buildscript {
+	repositories {
+		mavenCentral()
```
Comment (Mauin) [inline detekt-sample-extensions/build.gradle]: jCenter is a superset of mavenCentral. It should be safe to remove `mavenCentral()`

File: detekt-sample-extensions/build.gradle [hunk 2]
```diff
@@ -1,13 +1,40 @@
...
+		jcenter()
+	}
+
+	dependencies {
+		classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
+	}
+}
+
+repositories {
+	mavenCentral()
```
Comment (Mauin) [inline detekt-sample-extensions/build.gradle]: Same here

File: detekt-sample-extensions/build.gradle [hunk 3]
```diff
@@ -1,13 +1,40 @@
...
+	testImplementation "org.junit.jupiter:junit-jupiter-engine:$junitEngineVersion"
 
-	testImplementation project(':detekt-test')
 	testRuntimeOnly "org.junit.platform:junit-platform-launcher:$junitPlatformVersion"
 	testRuntimeOnly "org.junit.platform:junit-platform-console:$junitPlatformVersion"
 	testRuntimeOnly "org.jetbrains.spek:spek-junit-platform-engine:$spekVersion"
 }
+
+sourceSets {
+	main.java.srcDirs += 'src/main/kotlin'
```
Comment (Mauin) [inline detekt-sample-extensions/build.gradle]: Does the `kotlin` Gradle plugin not do this by default?

File: detekt-sample-extensions/build.gradle (hunk 1 above)
Comment (arturbosch) [reply detekt-sample-extensions/build.gradle]: Yep your right, build is still green :)

**Pull request merged**